### PR TITLE
CMR-4954 decouple access-control from echo in dev-system

### PR DESCRIPTION
**Pull request by daniel-zamora**
Remove logic relying on echo acls from CMR, instead rely on CMR acls.

### CODE REVIEW

File: umm-lib/src/cmr/umm/acl_matchers.clj [hunk 1]
```diff
@@ -37,22 +38,26 @@
       ;; Is end2 in the range
       (or (= end1 end2) (= start1 end2) (t/within? interval1 end2)))))
 
+(defn- parse-date
```
Comment (mschmele) [inline umm-lib/src/cmr/umm/acl_matchers.clj]: I feel like we would have a common-util function for this already. If not, this function would probably go very nicely there

Comment (daniel-zamora) [reply umm-lib/src/cmr/umm/acl_matchers.clj]: I took it from the other acl_matchers.clj

File: acl-lib/src/cmr/acl/acl_fetcher.clj [hunk 2]
```diff
@@ -72,12 +99,19 @@
...
   [context]
   (let [cache (cache/context->cache context acl-cache-key)
-        updated-acls (echo-acls/get-acls-by-types
+        updated-acls (process-search-for-acls
                        ;; All of the object identity types needed by the application are fetched. We want
                        ;; the cache to contain all of the acls needed.
                        context (context->cached-object-identity-types context))]
     (cache/set-value cache acl-cache-key updated-acls)))
 
+(comment
```
[the remark this answers is not in the document]
Comment (daniel-zamora) [reply acl-lib/src/cmr/acl/acl_fetcher.clj]: I did, just incase we need to debug its a pretty useful one.  I can remove it if you like.

[the remark this answers is not in the document]
Comment (mschmele) [reply acl-lib/src/cmr/acl/acl_fetcher.clj]: If it's useful then leave it. Maybe add a comment about what it's useful in testing if you're feeling saucy

**Pull request merged**